README.md

```diff
@@ -7,25 +7,45 @@
 
 ## Group Members
 
+1. Brent
+2. Coulter
+3. Jackson
+
 
 ## Project Description
 
+Building a web-app for users to connect with developers and pair program
+
 
 ## Who uses it?
 
+We will have 2 types of users. Students & Teachers
 
 ## What outputs do they need?
 
+Being connected to a chat app
+
 
 ## What inputs are needed to generate those outputs?
 
+The specific languages trying to be learned
+
 
 ## What technologies will you use that weren't covered in class?
 
+### Front End
+Meteor.js
+React + Redux
+
+### Back End
+golang
+
 
 ## What technologies do you plan to use?
 
+PostgreSQL
```

Comment:
Flesh out this list, even if it's noted elsewhere in the propoosal. You'll need to use a lot more tech than just PSQL.